### ISSUE TITLE
Can not download/build under Windows 10

### ISSUE THREAD

**Issue by KanybekMomukeyev** [Drakirus/go-flutter-desktop-embedder#36]
Hi, i can not download under windows 10
```
go run engineDownloader.go

Downloading file from
 'https://storage.googleapis.com/flutter_infra/flutter/72c7a7567228cdaf8b7aa4a9e3d212ef9d4cc0ed/windows-x64/windows-x64-embedder'
 to 'C:\src\flutter_projects\go-flutter-desktop-embedder\example\simpleDemo/.build/temp.zip'

Download completed in 1.5597002s3 [2K
Downloaded embedder for windows-x64 platform, matching version : 72c7a7567228cdaf8b7aa4a9e3d212ef9d4cc0ed
Downloading file from
 'https://storage.googleapis.com/flutter_infra/flutter/72c7a7567228cdaf8b7aa4a9e3d212ef9d4cc0ed/windows-x64/artifacts.zip'
 to 'C:\src\flutter_projects\go-flutter-desktop-embedder\example\simpleDemo/.build/artifacts.zip'

Download completed in 8.5188025s [2K
Downloaded artifact for windows-x64 platform.
2018/11/30 10:58:32 zip: not a valid zip file
exit status 1
````

**Comment by KanybekMomukeyev** [comment]
I tried follow the medium guide for this

https://medium.com/flutter-community/flutter-from-mobile-to-desktop-93635e8de64e

But have following issue
```
flutter: Observatory listening on http://127.0.0.1:50300/
flutter: тХРтХРтХб EXCEPTION CAUGHT BY WIDGETS LIBRARY тХЮтХРтХРтХРтХРтХРтХРтХРтХРтХРтХРтХРтХРтХРтХРтХРтХРтХРтХРтХРтХРтХРтХРтХРтХРтХРтХРтХРтХРтХРтХРтХРтХРтХРтХРтХРтХРтХРтХРтХРтХРтХРтХРтХРтХРтХРтХРтХРтХРтХРтХРтХРтХРтХРтХРтХРтХРтХРтХРтХР
flutter: The following NoSuchMethodError was thrown building IconTheme(IconThemeData#2abdc(color:
flutter: Color(0xdd000000))):
flutter: Class 'Window' has no instance getter 'locales'.
flutter: Receiver: Instance of 'Window'
flutter: Tried calling: locales
flutter:
flutter: When the exception was thrown, this was the stack:
flutter: #0      Object.noSuchMethod (dart:core/runtime/libobject_patch.dart:50:5)
flutter: #1      _WidgetsAppState.initState (package:flutter/src/widgets/app.dart:716:41)
flutter: #2      StatefulElement._firstBuild (package:flutter/src/widgets/framework.dart:3830:58)
flutter: #3      ComponentElement.mount (package:flutter/src/widgets/framework.dart:3696:5)
flutter: #4      Element.inflateWidget (package:flutter/src/widgets/framework.dart:2950:14)
flutter: #5      Element.updateChild (package:flutter/src/widgets/framework.dart:2753:12)
flutter: #6      ComponentElement.performRebuild (package:flutter/src/widgets/framework.dart:3732:16)
flutter: #7      Element.rebuild (package:flutter/src/widgets/framework.dart:3547:5)
flutter: #8      ComponentElement._firstBuild (package:flutter/src/widgets/framework.dart:3701:5)
flutter: #9      ComponentElement.mount (package:flutter/src/widgets/framework.dart:3696:5)
flutter: #10     Element.inflateWidget (package:flutter/src/widgets/framework.dart:2950:14)
flutter: #11     Element.updateChild (package:flutter/src/widgets/framework.dart:2753:12)
flutter: #12     ComponentElement.performRebuild (package:flutter/src/widgets/framework.dart:3732:16)
flutter: #13     Element.rebuild (package:flutter/src/widgets/framework.dart:3547:5)
flutter: #14     ComponentElement._firstBuild (package:flutter/src/widgets/framework.dart:3701:5)
flutter: #15     ComponentElement.mount (package:flutter/src/widgets/framework.dart:3696:5)
flutter: #16     Element.inflateWidget (package:flutter/src/widgets/framework.dart:2950:14)
flutter: #17     Element.updateChild (package:flutter/src/widgets/framework.dart:2753:12)
flutter: #18     ComponentElement.performRebuild (package:flutter/src/widgets/framework.dart:3732:16)
flutter: #19     Element.rebuild (package:flutter/src/widgets/framework.dart:3547:5)
flutter: #20     ComponentElement._firstBuild (package:flutter/src/widgets/framework.dart:3701:5)
flutter: #21     ComponentElement.mount (package:flutter/src/widgets/framework.dart:3696:5)
flutter: #22     Element.inflateWidget (package:flutter/src/widgets/framework.dart:2950:14)
flutter: #23     Element.updateChild (package:flutter/src/widgets/framework.dart:2753:12)
flutter: #24     ComponentElement.performRebuild (package:flutter/src/widgets/framework.dart:3732:16)
flutter: #25     Element.rebuild (package:flutter/src/widgets/framework.dart:3547:5)
flutter: #26     ComponentElement._firstBuild (package:flutter/src/widgets/framework.dart:3701:5)
flutter: #27     StatefulElement._firstBuild (package:flutter/src/widgets/framework.dart:3848:11)
flutter: #28     ComponentElement.mount (package:flutter/src/widgets/framework.dart:3696:5)
flutter: #29     Element.inflateWidget (package:flutter/src/widgets/framework.dart:2950:14)
flutter: #30     Element.updateChild (package:flutter/src/widgets/framework.dart:2753:12)
flutter: #31     ComponentElement.performRebuild (package:flutter/src/widgets/framework.dart:3732:16)
flutter: #32     Element.rebuild (package:flutter/src/widgets/framework.dart:3547:5)
flutter: #33     ComponentElement._firstBuild (package:flutter/src/widgets/framework.dart:3701:5)
flutter: #34     ComponentElement.mount (package:flutter/src/widgets/framework.dart:3696:5)
flutter: #35     Element.inflateWidget (package:flutter/src/widgets/framework.dart:2950:14)
flutter: #36     Element.updateChild (package:flutter/src/widgets/framework.dart:2753:12)
flutter: #37     ComponentElement.performRebuild (package:flutter/src/widgets/framework.dart:3732:16)
flutter: #38     Element.rebuild (package:flutter/src/widgets/framework.dart:3547:5)
flutter: #39     ComponentElement._firstBuild (package:flutter/src/widgets/framework.dart:3701:5)
flutter: #40     StatefulElement._firstBuild (package:flutter/src/widgets/framework.dart:3848:11)
flutter: #41     ComponentElement.mount (package:flutter/src/widgets/framework.dart:3696:5)
flutter: #42     Element.inflateWidget (package:flutter/src/widgets/framework.dart:2950:14)
flutter: #43     Element.updateChild (package:flutter/src/widgets/framework.dart:2753:12)
flutter: #44     ComponentElement.performRebuild (package:flutter/src/widgets/framework.dart:3732:16)
flutter: #45     Element.rebuild (package:flutter/src/widgets/framework.dart:3547:5)
flutter: #46     ComponentElement._firstBuild (package:flutter/src/widgets/framework.dart:3701:5)
flutter: #47     StatefulElement._firstBuild (package:flutter/src/widgets/framework.dart:3848:11)
flutter: #48     ComponentElement.mount (package:flutter/src/widgets/framework.dart:3696:5)
flutter: #49     Element.inflateWidget (package:flutter/src/widgets/framework.dart:2950:14)
flutter: #50     Element.updateChild (package:flutter/src/widgets/framework.dart:2753:12)
flutter: #51     RenderObjectToWidgetElement._rebuild (package:flutter/src/widgets/binding.dart:909:16)
flutter: #52     RenderObjectToWidgetElement.mount (package:flutter/src/widgets/binding.dart:880:5)
flutter: #53     RenderObjectToWidgetAdapter.attachToRenderTree.<anonymous closure> (package:flutter/src/widgets/binding.dart:826:17)
flutter: #54     BuildOwner.buildScope (package:flutter/src/widgets/framework.dart:2266:19)
flutter: #55     RenderObjectToWidgetAdapter.attachToRenderTree (package:flutter/src/widgets/binding.dart:825:13)
flutter: #56     _WidgetsFlutterBinding&BindingBase&GestureBinding&ServicesBinding&SchedulerBinding&PaintingBinding&SemanticsBinding&RendererBinding&WidgetsBinding.attachRootWidget (package:flutter/src/widgets/binding.dart:712:7)
flutter: #57     runApp (package:flutter/src/widgets/binding.dart:756:7)
flutter: #58     main (package:flutter_gallery/main.dart:26:3)
flutter: #59     _startIsolate.<anonymous closure> (dart:isolate/runtime/libisolate_patch.dart:289:19)
flutter: #60     _RawReceivePortImpl._handleMessage (dart:isolate/runtime/libisolate_patch.dart:171:12)
flutter: тХРтХРтХРтХРтХРтХРтХРтХРтХРтХРтХРтХРтХРтХРтХРтХРтХРтХРтХРтХРтХРтХРтХРтХРтХРтХРтХРтХРтХРтХРтХРтХРтХРтХРтХРтХРтХРтХРтХРтХРтХРтХРтХРтХРтХРтХРтХРтХРтХРтХРтХРтХРтХРтХРтХРтХРтХРтХРтХРтХРтХРтХРтХРтХРтХРтХРтХРтХРтХРтХРтХРтХРтХРтХРтХРтХРтХРтХРтХРтХРтХРтХРтХРтХРтХРтХРтХРтХРтХРтХРтХРтХРтХРтХРтХРтХРтХРтХРтХРтХР
```

**Comment by zephylac** [comment]
Can you put the result of your `flutter --version`

**Comment by Drakirus** [comment]
`go run engineDownloader.go` is fixed by #37.

Regarding:
```
flutter: The following NoSuchMethodError was thrown building IconTheme(IconThemeData#2abdc(color:
flutter: Color(0xdd000000))):
flutter: Class 'Window' has no instance getter 'locales'.
flutter: Receiver: Instance of 'Window'
flutter: Tried calling: locales
```

Using `Flutter 0.11.13 • channel beta` the *simpleDemo* does work.

Are you compiling your own flutter app ?
If so some tweaking might be needed!
